docs/api/app.md

```diff
@@ -782,27 +790,38 @@ starts:
 const {app} = require('electron')
 let myWindow = null
 
-const isSecondInstance = app.makeSingleInstance((commandLine, workingDirectory) => {
+const gotTheLock = app.requestSingleInstanceLock()
+
+app.on('second-instance', (commandLine, workingDirectory) => {
   // Someone tried to run a second instance, we should focus our window.
   if (myWindow) {
     if (myWindow.isMinimized()) myWindow.restore()
     myWindow.focus()
```

Comment:
Hey, not a blocker to this review (esp since it's not new to this diff) but what is the difference between `win.restore()` and `win.show()`?

It _looks_ like this block could be replaced with `if (myWindow) myWindow.show()` to restore + focus but it's not completely clear to me from the documentation, so I thought I'd ask...